_episodes/06-edges.md

```diff
@@ -28,7 +31,7 @@ Anecdotally, it is important to test edges cases because this is where errors te
 arise. Qualitatively different behavior happens at boundaries. As such,
 they tend to have special code dedicated to them in the implementation.
 
-> ## Consider the Fibonacci Sequence {.callout}
+> ## Consider a Fibonacci sequence
```

Comment:
There is only one Fibonacci sequence. It should remain "the Fibonacci sequence".